Common/Data/UniverseSelection/Universe.cs

```diff
@@ -170,6 +170,7 @@ public IEnumerable<Symbol> PerformSelection(DateTime utcTime, BaseDataCollection
         /// <param name="algorithm">The algorithm instance</param>
         /// <param name="marketHoursDatabase">The market hours database</param>
         /// <param name="symbolPropertiesDatabase">The symbol properties database</param>
+        /// <param name="accountCurrency">Base account currency</param>
```

Comment:
Docs added here (and elsewhere) but no matching parameter. Also, if we have access to the entire algorithm instance, then we can access `algorithm.Portfolio.CashBook.AccountCurrency` (to [Demeter's](https://en.wikipedia.org/wiki/Law_of_Demeter) demise of course :P )